Client/Frontend/Fakespot/Views/FakespotSettingsCardView.swift

```diff
@@ -103,6 +103,10 @@ final class FakespotSettingsCardView: UIView, ThemeApplicable {
         [showProductsLabel, recommendedProductsSwitch].forEach(labelSwitchStackView.addArrangedSubview)
         contentStackView.addArrangedSubview(labelSwitchStackView)
 
+        // FXIOS-7369: https://mozilla-hub.atlassian.net/browse/FXIOS-7369
+        labelSwitchStackView.isHidden = true
+        contentStackView.removeArrangedSubview(labelSwitchStackView)
```

Comment:
Wouldn't it make more sense to instead just not add `labelSwitchStackView` as an arrange subview? Adding it and then removing it again doesn't seem right.

Reply:
I didn't want to remove any code, but, yes, it's better to just not add the stack view

Reply:
You could just comment the line out and add an explaining comment on it.